controllers/user/index.js

```diff
@@ -74,7 +73,7 @@ router.get('/session', function (req, res) {
  * Public user routes
  * Disallow staff from this point on
  */
-router.use(requireNotStaffAuth, injectCopy('apply'), function (req, res, next) {
```

Comment:
This was the most egregious one, this was set here. Making `res.locals.copy` equal the `apply` locale key…which was then never used and consistently overridden with a `user.*` key in individual routers.